sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -51,13 +55,6 @@ shadowJar {
   }
 }
 
-test {
-  doFirst {
-    // Assert everything is in the output or test jar
-    assert classpath.size() == 2
```

Comment:
This is testing that we are actually testing the JDBC jar. You can not remove it.

Reply:
Gotcha, will add it back.